VERSION

```diff
@@ -0,0 +1 @@
+3.0.0
```

Comment:
Temporarily added VERSION file since this gem is included directly from the repository at this stage.
From several other repositories, i.e `conjurinc/appliance` and `cyberark/conjur`.
This will be removed before push to master

Reply:
With properly rebased downstream projects (conjur and appliance), and built via the MODE of RELEASE, the auto-generated VERSION file will be included in the resulting artifact, so there should be no need to have it here

Reply:
Yes, instead of consuming a gem, the repositories reference this repo.
Can the RELEASE mode with 0.0.0 argument produce a gem that can be referenced in other project Gemfile?

Reply:
RELEASE mode with a changelog entry of 0.0.0 will publish 0.0.0.pre.<jenkins build id> to rubygems as a gemfile.  Downstream projects would just need to add that version

Reply:
Thank you, I replaced the debify gem with 0.0.0.pre.47.